From 408788e26507026f12b82f3bd23b6b37393f5682 Mon Sep 17 00:00:00 2001
From: Thao Luong <thao.luong.uw@rvc.renesas.com>
Date: Wed, 27 May 2015 11:29:16 +0700
Subject: [PATCH 13/25] omxbufferpool: Don't set the offset in the memory when
 it is mapped

This patch removes the offset assignment overlapped with the one
specified in the GstMemory allocation.

refs #69206
Signed-off-by: Thao Luong <thao.luong.uw@rvc.renesas.com>
---
 omx/gstomxbufferpool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/omx/gstomxbufferpool.c b/omx/gstomxbufferpool.c
index 2eb2f67..652eadf 100644
--- a/omx/gstomxbufferpool.c
+++ b/omx/gstomxbufferpool.c
@@ -79,7 +79,7 @@ gst_omx_memory_map (GstMemory * mem, gsize maxsize, GstMapFlags flags)
 {
   GstOMXMemory *omem = (GstOMXMemory *) mem;
 
-  return omem->buf->omx_buf->pBuffer + omem->mem.offset;
+  return omem->buf->omx_buf->pBuffer;
 }
 
 static void
-- 
2.4.0

